src/test/java/org/sagebionetworks/bridge/models/schedules/MasterSchedulerConfigTest.java

```diff
@@ -0,0 +1,52 @@
+package org.sagebionetworks.bridge.models.schedules;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertSame;
+import static org.testng.Assert.assertTrue;
+
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+import org.sagebionetworks.bridge.models.TimestampHolder;
+import org.testng.annotations.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public class MasterSchedulerConfigTest {
```

Comment:
You don't need a separate test for MasterSchedulerConfigTest when it's identical to DynamoMasteSchedulerConfigTest. (Same with MasterSchedulerStatusTest.)